### ISSUE TITLE
Net Neutrality 

### ISSUE THREAD

**Issue by nhahuynh** [sketch-city/project-ideas#122]
The FCC has voted to begin dismantling net neutrality regulations. What's alarming is that this issue is relatively unknown outside of the IT circles. We need a way to explain net neutrality in layman terms and get the public mobilized. There are several orgs that are working on this like the ACLU and EFF so I don't want to just make another website tho. Thoughts?

**Comment by dylanjbarth** [comment]
@nhahuynh potentially similar to this idea https://github.com/sketch-city/project-ideas/issues/108 

what do you think?

**Comment by nhahuynh** [comment]
@dylanjbarth oh thanks! I didn't know that thread was related

**Comment by dylanjbarth** [comment]
@nhahuynh if you have any ideas on how that idea can be expanded it would be awesome to get your input there!
